docs/source/overview.rst

```diff
@@ -4,60 +4,16 @@
 Overview
 ********
 
-.. contents::
-    :local:
-    :depth: 2
-
-
-The Big Picture
-===============
-
-The following image gives an overview of the Twitcher components.
-
-.. image:: _images/twitcher-overview.png
-
-The Aim
-=======
-
-The aim is to have a simple to use security filter for OGC/OWS services (especially for Web Processing Services)
-which can be integrated in existing processing infrastructures.
-
-The design aims are:
-
-* Existing Web Processing Services and Clients should be able to use this security filter without modifications.
-* Simple: to use, to deploy, to maintain, to understand, ...
-* Token based security.
-* Tokens are only valid for a short period of time.
-* Able to be used in a distributed infrastructure (maybe using `Macaroons <https://github.com/rescrv/libmacaroons>`_?).
-* Python .. *ehm* not a requirement ... but more and more used.
-
 Twitcher Components
 ===================
 
 Twitcher consists of the following main parts:
 
 OWS Security
-   A wsgi middleware (actually currently it is a `Pyramid tween <http://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-tween>`_) which puts a simple token based security layer on top of a wsgi application. The access tokens are stored in a MongoDB.
+   A security layer to protect service access with x509 certificates and OAuth2 tokens.
```

Comment:
Maybe add more details about the scopes in a single location (list them and briefly tell where they apply).
I feel like the `register` and `compute` scopes required on different routes are not that obvious unless you look deeper in the code or scroll through every example.

Reply:
I have updated the api doc with some hints and referenced it.